modules/ROOT/pages/layers.adoc

```diff
@@ -39,11 +39,11 @@ If a node was not assigned to a layer at deployment time using the Runtime Manag
 . Select the application node to add to a layer. +
 You can hold the shift key to select multiple nodes to apply layers to all the selected nodes at once. +
 +
-_Result_: The *Customization* panel for the node appears in the right side of the panel as shown in the following image:
+_Result_: The *Customization* section for the node appears in *Details/Monitoring* card as shown in the following image:
```

Comment:
"appears in the Details tab of the node's card"

- we are going to be adding more tabs so we don't want to name for the card to keep expanding :)

Reply:
Accepted. Will change everywhere.